cmd/minikube/cmd/start.go

```diff
@@ -834,6 +834,16 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 		if runtime != "docker" {
 			out.WarningT("Using the '{{.runtime}}' runtime with the 'none' driver is an untested configuration!", out.V{"runtime": runtime})
 		}
+
+		conntrackK8sVersion, _ := semver.Make("1.18.0-rc.0")
```

Comment:
move this to one of our helper funcs, that checks against driver or k8s version.
validateDriver or ValidateFlag or one of those that has the right params

Reply:
this is already inside validateFlags. validateDriver doesn't have any more context than this function had, and this seemed like a natural fit since we're already checking for none-driver specific stuff.